### ISSUE TITLE
Lime3DS freezes after choosing a save file and starting in "A Link Between Worlds"

### ISSUE THREAD

**Issue by Mizuuta** [Lime3DS/Lime-3DS-Emulator#368]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

2117.1

### Description of Issue

After starting "A Link Between Worlds" and choosing a save file, that has already been played, and starting, the Lime3DS freezes.

Video:
https://github.com/user-attachments/assets/1445bee4-2aa8-426b-b6b1-f05468c7ee9b

### Expected Behavior

The game continues normally.

Video:
https://github.com/user-attachments/assets/9395d073-d424-416d-94f9-69c2438c4121

### Reproduction Steps

1. Start "A Link Between Worlds".
2. Select a save file, that already has been played. (My Savefile: [savefile.zip](https://github.com/user-attachments/files/16644801/savefile.zip))
3. Select "Start"
4. See it freeze

### Log File

[lime3ds_log.txt](https://github.com/user-attachments/files/16644793/lime3ds_log.txt)

### System Configuration

CPU: Intel i7-8700 (12) @ 4.600GHz
GPU/Driver: NVIDIA GeForce RTX 2060 Rev. A
RAM: 15923MiB
OS: Manjaro Linux x86_64

**Comment by DivXor** [comment]
In addition. Faced the same issue and found a temporary workaround that works for me.

Deleting this CEC (StreetPass) directory  `~/.var/app/io.github.lime3ds.Lime3DS/data/lime3ds-emu/nand/data/00000000000000000000000000000000/sysdata/00010026/00000000/CEC/000ec400/` fixes the issue (at least temporary) and allows to run a game save.

System Configuration
APU: 8 × AMD Custom APU 0932 (Steam Deck OLED)
OS: SteamOs 6.1.52-valve16-1-neptune-61 (64-bit)

**Comment by OpenSauce04** [comment]
Oh, that helps.
Did you import Streetpass data from outside of the emulator?
Attempting to use Streetpass data causes any game to crash. This is behaviour held over from Citra caused by a lack of Streetpass implementation.

**Comment by Mizuuta** [comment]
no, as far as i know i have not imported any Streetpass data.
(I have only imported my 'aes-keys.txt' and 'seeddb.bin' from my 3ds. don't know if that has something to do with that)
I just activated Streetpass in Game (I guess it must have created data there somehow).
Super Mario 3D Land also has Streetpass data, but doesn't crash.

**Comment by DivXor** [comment]
>  Did you import Streetpass data from outside of the emulator?

No, I didn't. It is a fresh Lime 2117 installation (from Flathub) and I haven't imported any data into the game.
